test/Entity/ClientEntityTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-authentication-oauth2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-authentication-oauth2/blob/master/LICENSE.md
+ *     New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Expressive\Authentication\OAuth2\Entity;
+
+use League\OAuth2\Server\Entities\ClientEntityInterface;
+use PHPUnit\Framework\TestCase;
+use Zend\Expressive\Authentication\OAuth2\Entity\ClientEntity;
+
+class ClientEntityTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->entity = new ClientEntity('foo', 'bar', 'http://localhost');
```

Comment:
I would write tests that ensure that the values passed to the constructor affected the state of the constructed entity.